packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -18,17 +18,23 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
 
 >**Note:** Okta implements SCIM 2.0 as specified in the RFC documents from the Internet Engineering Task Force:
 >
->- [Definitions, Overview, Concepts, and Requirements: RFC 7642](https://tools.ietf.org/html/rfc7642)
->- [Core Schema: RFC 7643](https://tools.ietf.org/html/rfc7643)
->- [Protocol: RFC 7644](https://tools.ietf.org/html/rfc7644)
+>* [Definitions, Overview, Concepts, and Requirements: RFC 7642](https://tools.ietf.org/html/rfc7642)
+>* [Core Schema: RFC 7643](https://tools.ietf.org/html/rfc7643)
+>* [Protocol: RFC 7644](https://tools.ietf.org/html/rfc7644)
 
 ## SCIM User operations
 
 ### Creating users
 
+![Flowchart - create user](/img/oin/scim_flow-user-create.png "Simple flow diagram for create user process")
+
 The User creation operation brings the user's application profile from Okta over to the Service Provider. A user's application profile represents the key-value attributes defined on the **Profile** tab when a user is added.
 
-To enable user provisioning, an Okta administrator must configure the provisioning options in the Okta Admin Console. In the Okta Admin Console, select your SCIM application from your list of applications. Under the **Provisioning** tab, click **To App** and **Edit**. In the **Create User** option, click **Enable** and then **Save**.
+To enable user provisioning, an Okta administrator must configure the provisioning options in the Okta Admin Console. In the Okta Admin Console:
```

Comment:
Who is the audience here? Aren't we talking directly to the Okta admin or no?

To enable user provisioning, you need to configure the provisioning options in the Okta Admin Console: